staging/src/k8s.io/apiserver/pkg/authentication/config/validation/validation.go

```diff
@@ -35,31 +35,25 @@ var (
 )
 
 // ValidateAuthenticationConfiguration validates a given AuthenticationConfiguration.
+//   - Zero JWT authenticators are allowed. This allows us to support the case where
+//     API server is initially configured with no authenticators and then authenticators are added
+//     later via API server restart or dynamic config reload.
```

Comment:
```suggestion
//   - Zero JWT authenticators are allowed. This allows us to support the case where the
//     API server is initially configured with no authenticators and then authenticators are added
//     later via dynamic config reload.
```

On API server restart this doesn't really matter because you would just set the flag if you needed to vs. not setting it at all.

Reply:
I added that because there was nothing blocking users from doing that as an option (restarting API server) but will remove it.